client/blocks/global-notice/README.md

```diff
@@ -0,0 +1,24 @@
+Global Notice
```

Comment:
It's different, the one you linked is plural and used to display all notices from the store. This one triggers a notice on mount, a bit like a data component.

I agree we should think how we organize the code as it can be confusing in the current state.